packages/database/scripts/setupNewDatabase.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$SCRIPT_DIR"
+cd ..
+
+if [ "$1" != "OK" ]; then
```

Comment:
Rather than checking whether they've passed in `OK`, should we check for the presence of `DB_NAME`? Is that what we're doing here? I note that `OK` is passed from package.json, where we explicitly set DB_NAME

Reply:
@rohan-bes do you have a response here?